src/CheckoutSdk/Reconciliation/Action.cs

```diff
@@ -7,11 +8,11 @@ public sealed class Action
     {
         public string Id { get; set; }
 
-        public string Type { get; set; }
+        public ActionType? Type { get; set; }
 
         public DateTime? ProcessedOn { get; set; }
 
-        public string ResponseCode { get; set; }
+        public long ResponseCode { get; set; }
```

Comment:
this field could be "long?" nullable?

Reply:
Nice catch! Yeah, will make it nullable.